assets/javascripts/src/components/pages/Payment.jsx

```diff
@@ -80,7 +80,17 @@ export default class Payment extends React.Component {
                             className="center-text"
                             outerClassName="half-width"
                             required/>
+
             </div>
+            {this.props.reducedCheckout && <InputField label="Email" id="email2" type="email" value={this.props.details.email}
+                                                      onChange={ event => this.props.updateDetails({ email: event.target.value })}
+                                                      required />}
+            {this.props.reducedCheckout && <div className="giraffe-checkbox">
```

Comment:
This would need both elements to share a root node though, which I'm not keen on.